my-account/app/com/commercetools/sunrise/myaccount/wishlist/SunriseRemoveFromWishlistController.java

```diff
@@ -0,0 +1,69 @@
+package com.commercetools.sunrise.myaccount.wishlist;
+
+import com.commercetools.sunrise.framework.controllers.SunriseContentFormController;
+import com.commercetools.sunrise.framework.controllers.WithFormFlow;
+import com.commercetools.sunrise.framework.hooks.EnableHooks;
+import com.commercetools.sunrise.framework.reverserouters.SunriseRoute;
+import com.commercetools.sunrise.framework.reverserouters.myaccount.wishlist.WishlistReverseRouter;
+import com.commercetools.sunrise.framework.template.engine.ContentRenderer;
+import com.commercetools.sunrise.myaccount.wishlist.viewmodels.WishlistLineItemFormData;
+import io.sphere.sdk.client.ClientErrorException;
+import io.sphere.sdk.shoppinglists.ShoppingList;
+import play.data.Form;
+import play.data.FormFactory;
+import play.libs.concurrent.HttpExecution;
+import play.mvc.Result;
+
+import javax.inject.Inject;
+import java.util.concurrent.CompletionStage;
+
+public class SunriseRemoveFromWishlistController extends SunriseContentFormController
+        implements WithFormFlow<ShoppingList, ShoppingList, WishlistLineItemFormData> {
+    private final WishlistLineItemFormData formData;
+    private final WishlistFinderBySession wishlistFinder;
+    private final RemoveFromWishlistControllerAction controllerAction;
+    private final WishlistReverseRouter reverseRouter;
+
+    @Inject
+    protected SunriseRemoveFromWishlistController(final ContentRenderer contentRenderer, final FormFactory formFactory, final WishlistLineItemFormData formData,
```

Comment:
Just leaving this comment here so I don't forget to think about whether it's better to have no associated template and rely on redirects for handling bad requests. As long as error messages appear the same it can certainly be better, and they rely on flash so it should work. But need to check.